src/components/beat.js

```diff
@@ -47,7 +47,7 @@ const MODELS = {
   mine: 'mineObjTemplate'
 };
 
-const WEAPON_COLORS = {right: 'blue', left: 'red'};
+const WEAPON_COLORS = { right: 'blue', left: 'red' };
```

Comment:
Can we remove spaces around brackets to keep style consistent?

Reply:
I think I'm using a different formatter, if you let me know which JS formatter was used I can re-format it correctly.

Reply:
I see. I don't use any auto formatter. On A-Frame we have a [linter pass before committing](https://github.com/aframevr/aframe/blob/master/package.json#L79) and usually correct there if needed. There's probably semistandar formatters there for your editor but never looked into it.

Reply:
Maybe we should have a linter in Moonrider now there are more people contributing

Reply:
I notice we do have a linter